docs/reference/modules/cluster/remote-clusters-connect.asciidoc

```diff
@@ -0,0 +1,219 @@
+[[remote-clusters-connect]]
+=== Connect to remote clusters
+Your local cluster uses the <<modules-network,transport layer>> to establish
+communication with remote clusters. The coordinating nodes in the local cluster
+establish <<long-lived-connections,long-lived>> TCP connections with specific
+nodes in the remote cluster. {es} requires these connections to remain open,
+even if the connections are idle for an extended period.
+
+NOTE: You must have the `manage` cluster privilege to connect remote clusters.
+
+To configure a remote cluster from Stack Management in {kib}:
+
+. Select *Remote Clusters* from the side navigation.
+. Specify the {es} endpoint URL, or the IP address or host name of the remote
+cluster followed by the transport port (defaults to `9300`). For example,
+`cluster.es.eastus2.staging.azure.foundit.no:9400` or `192.168.1.1:9300`.
+
+Alternatively, use the <<cluster-update-settings,cluster update settings API>>
+to add a remote cluster. You can also use this API to
+<<configure-remote-clusters-dynamic,apply and update>> settings across the
+entire remote cluster. To configure settings on individual nodes in a remote
+cluster, define
+<<configure-remote-clusters-static,static settings>> in `elasticsearch.yml`.
```

Comment:
This still reads funny to me. I think what we want to say is:

```suggestion
to add a remote cluster. You use this API to dynamically
<<configure-remote-clusters-dynamic,configure remote clusters>> for every node 
in the local cluster. To configure remote clusters for an individual node of the local cluster,
define <<configure-remote-clusters-static,static settings>> in the node's `elasticsearch.yml`.
```

Reply:
I'll change this to read:

> Alternatively, use the <<cluster-update-settings,cluster update settings API>>
> to add a remote cluster. You use this API to dynamically
> <<configure-remote-clusters-dynamic,configure remote clusters>> for _every_ node
> in the local cluster. To configure remote clusters for individual nodes in the
> local cluster, define <<configure-remote-clusters-static,static settings>> in `elasticsearch.yml` for each node.